[doc] Add KubeRay ecosystem

seanlaii opened the pull request:
* **Please check if the PR fulfills these requirements**
- [ ] The commit message follows our guidelines


* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
/kind documentation

* **What this PR does / why we need it**:
KubeRay has integrated with Volcano to support [gang-scheduling](https://docs.ray.io/en/latest/cluster/kubernetes/k8s-ecosystem/volcano.html#kuberay-integration-with-volcano) and [network topology aware scheduling](https://github.com/ray-project/kuberay/pull/4105) features.
We would like to a guideline in volcano's documentation.

* **Which issue(s) this PR fixes**:
https://github.com/ray-project/kuberay/issues/4289

JesseStutler commented:
@seanlaii Does this pr have rendering problem? I see that all the netlify CI failed

JesseStutler commented on content/en/docs/kuberay_on_volcano.md:
````diff
@@ -0,0 +1,124 @@
+# test-cluster-0-head-jj9bg            1/1     Running   0          36s
+```
+
+##### RayJob Example
````
Could you also add vcjob example? https://github.com/volcano-sh/volcano/blob/master/docs/user-guide/how_to_use_ray_plugin.md, in volcano v1.13 version, we have also supported using vcjob to create raycluster

win5923 commented on content/en/docs/ray_on_volcano.md:
```diff
@@ -0,0 +1,219 @@
+| Label | Description | Required |
+|-------|-------------|----------|
+| `ray.io/priority-class-name` | Assigns a [Kubernetes](https://kubernetes.io/docs/concepts/scheduling-eviction/pod-priority-preemption/#priorityclass) priority class for pod scheduling | No |
+| `volcano.sh/queue-name` | Specifies the Volcano queue for resource submission | No |
```
```suggestion
```

seanlaii replied:
Could you elaborate on this? I think we still support the queue name IIRC. Thanks!

win5923 replied:
Sorry, this was my mistake. just ignore it.

win5923 reviewed (approved):
LGTM for the KubeRay part, thanks for your effort!

seanlaii commented:
@win5923 Thanks for the review!
@JesseStutler I have updated the doc and added the Chinese version. Please take a look when you have a chance. Thanks!
